lib/rules/headers/errata.js

```diff
@@ -6,11 +6,27 @@ const self = {
 
 export const { name } = self;
 
+// Check if document is Recommendation, and uses inline changes(REC with Candidate/Proposed changes)
+function isRECWithChanges(sr) {
+    const isREC = sr.config.status === 'REC';
+    if (!isREC) {
+        return false;
+    }
+
+    const recMeta = sr.getRecMetadata({});
+    return Object.values(recMeta).some(v => !!v);
```

Comment:
it's probably faster to check the size of the array.
```suggestion
    return Object.values(recMeta).length !== 0;
```